src/pages/PrintPage.tsx

```diff
@@ -17,22 +17,22 @@ interface State {
   showConfirmModal: boolean
 }
 
-class SummaryPage extends React.Component<RouteComponentProps, State> {
+class PrintPage extends React.Component<RouteComponentProps, State> {
   public static contextType = BallotContext
   public state: State = {
     showConfirmModal: false,
   }
   public componentDidMount = () => {
-    window.addEventListener('afterprint', this.resetBallot)
+    window.addEventListener('afterprint', this.afterPrint)
   }
   public componentWillUnmount = () => {
-    window.removeEventListener('afterprint', this.resetBallot)
+    window.removeEventListener('afterprint', this.afterPrint)
   }
-  public resetBallot = () => {
+  public afterPrint = () => {
     // setTimeout to prevent a React infinite recursion issue
     window.setTimeout(() => {
       this.context.incrementBallotsPrintedCount()
-      this.context.resetBallot('/cast')
+      this.context.resetBallot('/')
```

Comment:
Why do we need to do this, it feels better to go `/cast` for the screen and then not use the state anymore to alter what `/` does?

Reply:
It's not possible to use the router to direct based on the url because there is no router loaded in this state. This was the workaround which we decided on late night last week. I had simply forgot to remove the route. This routing is completely useless at this point, hence I was removing it.

Reply:
Right, I'm wondering if the solution from last week is no longer the way we want to go given this new approach of no longer polling. I'm wondering if we should direct to `/cast` and render the fourth interstitial there.

Reply:
Removing the `/cast` url is simply clean up from the previous changes.

In order to render at a specific URL we would need to add a second router (similar to what is in `Ballot.tsx`) and then manage paths (of which there would only be `/` and `/cast` at this point).

Thus the solution you are suggesting involves more work which is not worth it until we need to navigate multiple pages in this at different URLs in this "logged out" state.

Any change to the strategy should be captured in a new issue, no?

Reply:
I can see your point, though I still worry about having `/` do anything other than start the election. I can live with where we are today.